scripts/populate_new_and_noteworthy_projects.py

```diff
@@ -0,0 +1,112 @@
+"""
+This will update node links on POPULAR_LINKS_NODE and NEW_AND_NOTEWORTHY_LINKS_NODE.
+"""
+import sys
+import logging
+import datetime
+import dateutil
+import operator
+import requests
+from modularodm import Q
+from website.app import init_app
+from website import models
+from framework.auth.core import Auth
+from scripts import utils as script_utils
+from framework.mongo import database as db
+from framework.transactions.context import TokuTransaction
+from website.project.model import Pointer
+from website.settings import POPULAR_LINKS_NODE, NEW_AND_NOTEWORTHY_LINKS_NODE, NEW_AND_NOTEWORTHY_CONTRIBUTOR_BLACKLIST, DOMAIN
+
+logger = logging.getLogger(__name__)
+
+def retrieve_data(url):
+    """ Fetch data and decode json """
+    response = requests.get(url)
+    data = response.json()
+    return data
+
+def get_popular_nodes():
+    """ Fetch data from url that returns dict with a list of popular nodes from piwik """
+    discover_url = DOMAIN + 'api/v1/explore/activity/popular/raw/'
```

Comment:
Ah, looks like we do store popular projects in piwik. In that case, you should still be able to call the ` popular_activity_json` function directly rather than sending a request to APIv1.